macros/loop_vars.sql

```diff
@@ -0,0 +1,24 @@
+{% macro loop_vars(vars) %}
```

Comment:
Just out of curiosity is there an advantage to using this method vs. our `select_from_values` macro?

Reply:
`select_from_values` is a macro used to efficiently load quite a lot of data into different data warehouses, because doing many `select x union all select y` behaves poorly after hundreds/thousands of statements.

on the other hand, `loop_vars` is looping through vars and just loading with the typical `select x union all select y` because performance is not an issue

So the 2 macros are just doing different things, hence why we need 2 😄